tools/sync_voodoo_palette.py

```
@@ -0,0 +1,269 @@
+
+    Returns:
+        the updated contents
+    """
+    start = source.find(BEGIN_MARKER)
+    end = source.find(END_MARKER)
+    if start < 0 or end < 0:
+        raise SystemExit(
+            "Could not find the generated palette markers in '%s'. Restore "
+            "them around DEFAULT_APP_COLOR_POOL:\n\n%s\n"
+            "DEFAULT_APP_COLOR_POOL = [...]\n%s"
+            % (CONSTANTS_PATH, BEGIN_MARKER, END_MARKER)
```

Ruff (0.16.1)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)